ui/app/scripts/controllers/newDeploymentCtrl.js

```diff
@@ -12,7 +12,7 @@ angular.module('apollo')
         var previouseEnvironmentLocalStorageKey = 'previous-run-environment-id';
 
 
-        var favoriteServicesLocalStorageKey = 'favorite-services-names';
+        var favoriteServicesLocalStorageKey = 'favorite-servicesMock-names';
```

Comment:
you going to revert all those mock after connect to the real data?

Reply:
its in the old ui folder please revert if you don't need it

Reply:
I don't know why it happened, maybe I refactored a name using the WebStorm shortcuts and it was affected. I'll revert.